plenum/test/stasher.py

```diff
@@ -132,17 +132,17 @@ def reset_delays_and_process_delayeds(self, *names):
 
 
 @contextmanager
-def delay_rules(stasher, *delayers):
+def delay_rules(node, *delayers):
```

Comment:
This context manager was taking stasher and not node because there are other stashers besides `nodeIbStasher`, like `clientIbStasher` and `actionQueueStasher`. This context manager is used only with `nodeIbStasher` now, but it can change in future. Besides there are other tests that use `delay_rules` and they will be broken by this change.